tests/credential_plugins_test.py

```diff
@@ -162,3 +165,89 @@ def test_tss_import(self) -> None:
         ):
             # assert this module as opposed to older thycotic.secrets.server
             assert cls.__module__ == 'delinea.secrets.server'
+
+
+@pytest.mark.parametrize(
+    (
+        'url',
+        'app_id',
+        'object_query',
+        'object_query_format',
+        'reason',
+        'verify',
```

Comment:
Out of all these, only `reason` differs. So I don't see why they should be duplicated and declared as params.

Parametrizing just `reason` would be equivalent and more than enough.

Although, looking into the rest of the commit, I see that it makes sense to add two other params. But only for things that actually differ.